### DISCUSSION TITLE
[WASM] Theming not effective for ContentDialog?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#11317] [Q&A]
I have a custom theme in place with works fine for WindowsAppSDK and WASM. 

However, it appears to be ineffective for `ContentDialog` on WASM: `ContentDialog` still shows with the grey default background etc.

Am I missing something or is this expected?

**Comment by jeromelaban** [comment]
Thanks for the report. If you can provide a repro if will be useful to understand what you're trying to do.

**Reply by DierkDroth** [reply]
Thanks @jeromelaban. Basically ContentDialog did not pick up the custom styling - which was a variation [of the style here](https://github.com/unoplatform/Uno.Themes/blob/master/src/library/Uno.Material/Styles/Controls/v2/ContentDialog.xaml) - on WASM, but did so on WindowsAppSDK (same XAML code).

Anyway, since I got sick of the various limitations and issues (some are [mentioned here](https://platform.uno/docs/articles/guides/silverlight-migration/07-dialogs.html)) imposed by the WindowsAppSDK ContentDialog implementation I went ahead this AM and implemented my own custom ContentDialog. Basically, a transparent 'overlay' over the application window + ThemeShadow/EvelatedView does the trick, which works for WASM and WindowsAppSDK.

FWIW I attached a code snippet in case anyone is interested

[UnusualContentDialog.zip](https://github.com/unoplatform/uno/files/10744326/UnusualContentDialog.zip)
